tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -0,0 +1,258 @@
+import csv
+import datetime
+import json
+import zoneinfo
+
+import pytest
+
+from sds_data_manager.lambda_images.instruments.batch_starter import (
+    all_dependency_present,
+    get_filename_from_event,
+    get_process_details,
+    prepare_data,
+    query_dependencies,
+    query_dependents,
+    remove_ingested,
+)
+
+
+@pytest.fixture(scope="session")
+def mock_event():
+    """Example of the type of event that will be passed to
+    the instrument lambda (in our case batch_starter.py).
+    """
+    with open("../test-data/codicehi_event.json") as file:
+        event = json.load(file)
+    return event
+
+
+@pytest.fixture()
+def setup_database(postgresql):
+    """Populate test database."""
+
+    cursor = postgresql.cursor()
+    cursor.execute("CREATE SCHEMA IF NOT EXISTS sdc;")
+
+    # Drop the table if it exists, to start with a fresh table
+    cursor.execute("DROP TABLE IF EXISTS sdc.codicehi;")
+
+    sql_command = """
```

Comment:
Is this going to be used in the RDS code as well? It seems like it should be abstracted out into the code somehow so that you are initializing your test DB table with the same schema as the real one and that they don't get out of sync here and your tests are still passing even though you updated your code schema.